### DISCUSSION TITLE
Preserve empty lines

### DISCUSSION THREAD

**Discussion by francisashley** [nuxt/content#1968] [Q&A]
Hi, I am looking to preserve all empty lines in markdown documents. 

I tried both disabling the `remark-squeeze-paragraphs` plugin in the configuration file

```
  content: {
    markdown: {
      remarkPlugins: {
        'remark-squeeze-paragraphs': false,
      },
    },
  },
```
and deleting all reference to `rehype`/`remark` in the dist source code.

In both cases, all empty lines were still cleared from `body.children` on `http://localhost:2015/api/_content/query`.

Anyone have a suggestion on how to preserve these? Thanks

**Comment by ctxinf** [comment]
samiliar needs, blank lines at the bottom of a code block will not be rendered, but in some scenerio like txt content display require blank lines in the end.

**Comment by Codaea** [comment]
I encountered a similar issue and worked around it by modifying the [Prose Paragraph](https://content.nuxt.com/components/prose#prosep)  component to include a `<br>` tag, like this:

```vue components/content/ProseP.vue
<template>
  <p><slot /></p>
  <br>
</template>
```

This approach may not work in all cases and doesn't directly address the original problem, but it could be a potential solution in some scenarios where squeezing all of text together is not ideal.
